src/applications/search/actions/index.js

```diff
@@ -15,6 +16,11 @@ export function fetchSearchResults(query, page, options) {
     if (page) {
       queryString = queryString.concat(`&page=${page}`);
     }
+    if (!query) {
```

Comment:
I think instead of dispatching a new action for when there's no search results (the request was still technically successful), you could change [this line](https://github.com/department-of-veterans-affairs/vets-website/pull/17818/files#diff-694c2df9019ca4d1e3bc85fb729b2e30f6660c3abe0e0420fd7ec49a3d488155R47) to use `results: response?.data?.attributes?.body || []` 🙂 that way we don't have to add another action to manage

Reply:
the fetch request actually fails without a valid query, so thats why I created this short circuit action to bypass the api request

Reply:
oh interesting! could you use this then instead?

```js
dispatch({
  type: FETCH_SEARCH_RESULTS_FAILURE,
  errors: [error],
});
```

Reply:
Alternatively if you don't want an error to show, you could do something like this:
```js
dispatch({
  type: FETCH_SEARCH_RESULTS_SUCCESS,
  results: [],
  meta: undefined,
});
```

Reply:
@kelsonic , after looking at this I think I want to stick with the new action plan. We don't want to throw an error, and using the FETCH_SEARCH_RESULTS_SUCCESS requires a lot of updates to that action to handle it being passed an empty results array and an undefined meta. I think the current solution is the cleanest